Form-Controls/index.html

```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label> Name
+      <input required type="text" placeholder="Bereket Hayelom">
+      </label>
+  </div>
+  <br>
+  <div> 
+      <label> Email
+      <input required type="email" placeholder="bereket@example.com">
+      </label>
+  </div>
+  <br>
+  <div> 
+    <label> color
+    <select name="tshirt-color" id="tshirt-color">
+      <option value="">black</option>
+      <option value="">green</option>
+      <option value="">blue</option>
```

Comment:
What does the `value` prop do for select inputs? Same question for the size input